packages/shared/src/hooks/utils/useShortcuts.ts

```diff
@@ -11,13 +15,15 @@ interface UseShortcuts {
  * Keywords: shortcutsUI, onboarding_most_visited
  */
 export const useShortcuts = (): UseShortcuts => {
-  const { user } = useAuthContext();
+  const { isLoggedIn, user } = useAuthContext();
+  const { value: shortcutsUIVersion } = useConditionalFeature({
+    feature: feature.shortcutsUI,
+    shouldEvaluate: isLoggedIn && isExtension,
```

Comment:
but won't this "re-set" people who already where proven to have the experiment as a win?
Not sure how to fix it, but something as in everyone after set date should also ALWAYS have it right?

Reply:
That could be part of enrolment from GB

Reply:
In that case we need to be very explicit towards experiment owners :) 
So yeah maybe let's raise the experiment on slack and see what will be advised